app/utils/commons.js

```diff
@@ -0,0 +1,7 @@
+export const isIterable = (obj) => {
+  // checks for null and undefined
+  if (obj == null) {
+    return false;
+  }
+  return typeof obj[Symbol.iterator] === 'function';
```

Comment:
This should be able to be simplified to:

```javascript
export const isIterable = (obj) => obj && typeof obj[Symbol.iterator] === 'function';
```

I also understand if you like the clarity of explicitly checking for null, though.

Reply:
It is just `one line` version isn't it?

Reply:
Make it 2 or 3 lines so it doesn't break 80 chars/line rules

Reply:
I did, see my code. It already break into lines.

Reply:
```javascript
export const isIterable = (obj) =>
  obj && typeof obj[Symbol.iterator] === 'function';
```